proxy/src/http/prom.rs

```diff
@@ -43,59 +44,36 @@ use crate::{
     error::{build_ok_header, ErrNoCause, ErrWithCause, Error, Internal, InternalNoCause, Result},
     execute_plan,
     forward::ForwardResult,
-    grpc::write::{execute_insert_plan, WriteContext},
-    Proxy,
+    Context as ProxyContext, Proxy,
 };
 
 impl reject::Reject for Error {}
 
 impl<Q: QueryExecutor + 'static> Proxy<Q> {
     /// Handle write samples to remote storage with remote storage protocol.
     async fn handle_prom_write(&self, ctx: RequestContext, req: WriteRequest) -> Result<()> {
```

Comment:
InfluxDB write should also updated.

Reply:
I will refactor all the writing process in another pr, and will not modify it in this pr for now.